physics/cu_gf_driver_post.F90

```diff
@@ -66,20 +66,19 @@ subroutine cu_gf_driver_post_run (im, km, t, q, prevst, prevsq, cactiv, cactiv_m
           conv_act_m(i)=0.0
         endif
         ! reflectivity parameterization for parameterized convection (reference:Unipost MDLFLD.f)
-        if(sqrt(garea(i)).lt.6500.)then
         ze      = 0.0
         ze_conv = 0.0
         dbz_sum = 0.0
-        cuprate = raincv(i) * 3600.0 / dt          ! cu precip rate (mm/h)
-        ze_conv = 300.0 * cuprate**1.4
-        if (maxupmf(i).gt.0.05) then
+        cuprate = max(0.1,1.e3*raincv(i) * 3600.0 / dt)          ! cu precip rate (mm/h)
+        if(cuprate .lt. 0.05) cuprate=0.
+        ze_conv = 300.0 * cuprate**1.5
+        if (maxupmf(i).gt.0.1 .and. cuprate.gt.0.) then
```

Comment:
Is not cuprate always >0 after adjustment? Any reason to change cuprate and add the condition of cuprate>0?

Reply:
> Is not cuprate always >0 after adjustment? Any reason to change cuprate and add the condition of cuprate>0?

In line 73, If the convective precipitation rate is very weak (<0.05 mm/h), the cuprate is reset to 0. Thus in the if condition of line 75, we exclude the case with weak convective updraft and weak convective precipitation.

Reply:
The problem is in line 72, cuprate will be always >0.1 mm/h.